src/zac/accounts/views.py

```diff
@@ -42,23 +54,101 @@ class PermissionSetsView(LoginRequiredMixin, ListView):
     queryset = PermissionSet.objects.all()
 
 
-class PermissionSetCreateView(PermissionRequiredMixin, CreateView):
+class PermissionSetCreateView(
+    PermissionRequiredMixin, InformatieobjecttypeFormsetMixin, CreateView
+):
     model = PermissionSet
     form_class = PermissionSetForm
     permission_required = "accounts.add_permissionset"
     success_url = reverse_lazy("accounts:authprofile-list")
 
+    def get(self, request, *args, **kwargs):
+        self.object = None
+        informatieobjecttype_formset = self.construct_formset()
+        permissionset_form = self.get_form(self.form_class)
+        return self.render_to_response(
+            self.get_context_data(
+                form=permissionset_form,
+                informatieobjecttype_formset=informatieobjecttype_formset,
+            )
+        )
+
+    @transaction.atomic()
+    def post(self, request, *args, **kwargs):
+        self.object = None
+        permissionset_form = self.get_form(self.form_class)
+        informatieobjecttype_formset = self.construct_formset()
+
+        if (
+            not permissionset_form.is_valid()
+            or not informatieobjecttype_formset.is_valid()
+        ):
+            return self.form_invalid(permissionset_form, informatieobjecttype_formset)
+
+        return self.form_valid(permissionset_form, informatieobjecttype_formset)
+
 
 class PermissionSetDetailView(LoginRequiredMixin, DetailView):
     queryset = PermissionSet.objects.prefetch_related("authorizationprofile_set")
     context_object_name = "permission_set"
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        informatieobjecttype_permissions = (
+            InformatieobjecttypePermission.objects.filter(
+                permission_set=context["object"]
```

Comment:
```suggestion
                permission_set=self.object
```
`self.object` is set at the beginning of the `def get` in `DetailView`. It's the result of `self.get_object()`. Pulling stuff from the context while it's available as a view attribute seems a bit backwards.